py-rse/errors.Rmd

```diff
@@ -28,14 +28,16 @@ for denom in [-5, 0, 5]:
 
 `try`/`except` looks like `if`/`else` and works in a similar fashion.
 If nothing unexpected happens inside the `try` block,
-the `except` block isn't run.
+the `except` block isn't run (Figure \@ref(fig:py-rse-errors-control-flow)).
 If something *does* happen inside the `try`,
 the program jumps immediately to the `except`.
 This is why the `print` statement inside the `try` doesn't run when `denom` is 0:
 as soon as Python tries to calculate `1/denom`,
 it skips directly to the error message.
 
-FIXME: diagram of exception handling control flow
+```{r py-rse-errors-control-flow, echo=FALSE, fig.cap="Exception Control Flow"}
+knitr::include_graphics("figures/FIXME.png")
+```
```

Comment:
Can you provide a sketch of this figure?

Reply:
Added.